pkg/networkpath/traceroute/runner.go

```diff
@@ -81,6 +98,24 @@ func RunTraceroute(cfg Config) (NetworkPath, error) {
 	return pathResult, nil
 }
 
+func getPorts(configDestPort uint16) (uint16, uint16, bool) {
+	var destPort uint16
+	var srcPort uint16
+	var useSourcePort bool
+	if configDestPort > 0 {
+		// Fixed Destination Port + Random Source Port
+		destPort = configDestPort
+		useSourcePort = true
+		srcPort = DefaultSourcePort + uint16(rand.Intn(10000))
+	} else {
+		// Random Destination Port + Random Source Port
+		destPort = DefaultDestPort + uint16(rand.Intn(30))
+		useSourcePort = false
+		srcPort = DefaultSourcePort + uint16(rand.Intn(10000))
```

Comment:
Seems like this line can be moved out of the if/else.

Reply:
Good catch, the two lines were different previously, but I changed them later and they are indeed same now.

Fixed here https://github.com/DataDog/datadog-agent/pull/23814/commits/865665b5827571c31c1eaa31df69c7608693d356